src/core/render-biblio.js

```diff
@@ -4,9 +4,33 @@
 import { addId } from "./utils";
 import hyperHTML from "hyperhtml";
 import { pub } from "./pubsubhub";
+import { lang as defaultLang } from "../core/l10n"
 
 export const name = "core/render-biblio";
 
+const localization_strings = {
+  en: {
+    info_references: "Informative references",
+    norm_references: "Normative references",
+    references: "References"
+  },
+  nl:  {
+    info_references: "Informatieve referenties",
+    norm_references: "Normatieve referenties",
+    references: "Referenties"
+  },
+  es: {
+    info_references: "Referencias informativas",
+    norm_references: "Referencias normativas",
+    references: "Referencias"
+  }
+}
+
+const lang = defaultLang in meta ? defaultLang : "en";
```

Comment:
I think here you want: 

```suggestion
const lang = defaultLang in localizationStrings ? defaultLang : "en";
```

Reply:
Oops! I totally skipped. Will change this now